### PR TITLE
("add book detail and functions on clic")

### PR DESCRIPTION
**Pull request by jsequea**
Add new function an files book detail, js, styles and function for redirect path.

image

### CODE REVIEW

File: styles/styles-book-detail.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    font-size: 20px;
+    line-height: 24px;
+    font-weight: 300;
+  }
+
+  .portada .cover{
+    width: 261px;
+    height: 368px;
+  }
+
```
Comment (dsandoval11) [inline styles/styles-book-detail.js]: Don't mix English and Spanish, fix all styles 😬

File: styles/styles-book-detail.js [hunk 2]
```diff
@@ -0,0 +1,192 @@
+import {css} from "lit-element";
+
+export default css`
+
+@import 'styles';
```
Comment (dsandoval11) [inline styles/styles-book-detail.js]: This doesn't work because it's not sass

Comment (jsequea) [reply styles/styles-book-detail.js]: fixes changes

File: styles/styles-book-detail.js (hunk 1 above)
Comment (jsequea) [reply styles/styles-book-detail.js]: language fixes successful

**Review by dsandoval11 — APPROVED** [review]

**Pull request merged**